src/docs/user/references/DataConnectors.adoc

```diff
@@ -234,64 +234,85 @@ In the import mode, the output containing the imported data takes one or many of
 * in a _JSON_ format using the variable `DATAFRAME_JSON`.
 
 ==== Database Connections Pooling
-To allow the persistence of a single connection across multiple SQL tasks (basically when running several Tasks concurrently on the same machine),
+To allow the persistence of a single connection across multiple SQL tasks (basically when running several tasks concurrently on the same machine),
 we have implemented the connection pooling feature to any external SQL database based on the https://github.com/brettwooldridge/HikariCP[HikariCP^] Framework.
-This feature is showcased through two workflow templates: SQL_Pooled_Connection_Query and SQL_Pooled_Connection_Update.
-The SQL_Pooled_Connection_Query workflow is based on the replicate control flow where the split task is Open_Pooled_Connection, the process task is Execute_Query_Statements and finally the merge task is Merge_Result_Set.
-The SQL_Pooled_Connection_Update workflow is composed of two tasks: Open_Pooled_Connection and Execute_Update_Statements.
+This feature is showcased through two workflow templates: SQL_Pooled_Connection_Query and SQL_Pooled_Connection_Update. Both are a single task workflow.
 
 *Variables:*
 
-Both workflows share the same list of variables. Consequently, we describe them in the same following table.
+The variables listed in the following table exist for both the SQL_Pooled_Connection_Update and the SQL_Pooled_Connection_Query workflows.
```

Comment:
for both, the SQL_Pooled_Connection_Update and the SQL_Pooled_Connection_Query workflows.
OR
for both SQL_Pooled_Connection_Update and SQL_Pooled_Connection_Query workflows.

Up to you.